Jenkinsfile

```diff
@@ -14,66 +14,69 @@ elifePipeline {
+        //     }
+        //     elifeMainlineOnly {
+        //         folder = 'develop'
+        //     }
+        //     actions['styleguide'] = {
+        //         def targetUrl = "https://s3.amazonaws.com/ci-elife-xpub-styleguide/${folder}/index.html"
+        //         withCommitStatus(
+        //             {
+        //                 try {
+        //                     sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --name elife-xpub_app_style_guide app npm run build:styleguide"
+        //                     sh "docker cp elife-xpub_app_style_guide:/home/xpub/_build_styleguide ${folder}"
+        //                     sh "aws s3 cp --recursive ${folder} s3://ci-elife-xpub-styleguide/${folder}"
+        //                     echo "Styleguide URL: $targetUrl"
+        //                 } catch (e) {
+        //                     sh "docker-compose -f docker-compose.yml -f docker-compose.ci.yml down -v"
+        //                 }
+        //             },
+        //             [
+        //                 'name': 'styleguide',
+        //                 'commit': commit,
+        //                 'targetUrl': targetUrl,
+        //             ]
+        //         )
+        //     }
 
-            try {
-                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d postgres"
-                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_wait_postgres app bash -c './scripts/wait-for-it.sh postgres:5432'"
-                parallel actions
-            } finally {
-                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml down -v"
-            }
-        }
+        //     try {
+        //         sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d postgres"
+        //         sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_wait_postgres app bash -c './scripts/wait-for-it.sh postgres:5432'"
+        //         parallel actions
+        //     } finally {
+        //         sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml down -v"
+        //     }
+        // }
 
         stage 'Browser Tests', {
             try {
                 sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d postgres"
                 sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_wait_postgres app bash -c './scripts/wait-for-it.sh postgres:5432'"
+                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml run --rm --name elife-xpub_clobber_db app bash -c 'npx pubsweet setupdb --clobber --username test --password password --email test@example.com'"
```

Comment:
The environment variables
```
NODE_ENV=production NODE_CONFIG_ENV=test PGDATABASE=test_browser
```
(from the `test:browser` npm targets 3 lines later) are probably necessary in these commands too, because they set up a different `test_browser` database that should be clobbered and used to run the application in `docker--compose up`.

Reply:
I think this is a hang-over from when the unit tests were run in parallel with the browser tests - so we should take this env var out.

Reply:
We've also ensured all commands are passed the same env vars ... done!